### ISSUE TITLE
segfault on get_bucket

### ISSUE THREAD

**Issue by jeffmichels** [boto/boto#3148]
I've got python 2.7.6 installed on OS X Yosemite and am getting a seg fault right out the gate with a simple attempt to get_bucket.  Googling around I don't see anyone else with the same issue.  Any thoughts on this.

$ python
Python 2.7.6 (default, Sep  9 2014, 15:04:36) 
[GCC 4.2.1 Compatible Apple LLVM 6.0 (clang-600.0.39)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import boto
>>> c  = boto.connect_s3()
>>> b = c.get_bucket('mybucket')
Segmentation fault: 11
$

I believe I have the latest boto (2.38.0)

thanks

-Jeff

**Comment by grahamlyons** [comment]
Have you seen any recurrence of this in later versions, @jeffmichels? If not then I suggest we close this for now.

**Comment by jeffmichels** [comment]
no.  this was for a project that I haven't contributed to in a while.  seems to be working now though.
